src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -173,23 +173,13 @@
       (style/create-section-header "Analysis Submissions")
       (style/create-paragraph
         (let [count-all (count submissions)
-              success-count (->> submissions (filter all-success?) count)
```

Comment:
OK.  That is useful information, and can be seen in the Workflows section of the Monitor tab.  But based on this story and a conversation with David Siedzik I don't think it belongs *here*.  Here we want to match the Submissions section of the Monitor tab.